src/Components/Calculator.jsx

```diff
@@ -1,62 +1,56 @@
-import React from 'react'
+import { useState } from 'react'
 import Button from './Button'
 import buttons from '../assets/calculatorKeys'
 import calculate from '../Logic/calculate'
-class Calculator extends React.Component {
-	constructor(props) {
-		super(props)
+function Calculator(){
 
```

Comment:
- [ ] Kindly use arrow function instead of conventional functions to follow ES6 😊

Your Function 🔴
```
function Calculator(){
// your contents
}
```

Arrow Function 🟢
```
const Calculator = () => {
// your contents
}
```